requirements.txt

```diff
@@ -1,4 +1,4 @@
-pynacl>=1.4.0
-discord.py>=1.7.3
-tinytag==1.7.0
+nextcord[voice]>=2.0.0a6
+nextcord[speed]>=2.0.0a6
```

Comment:
(note that this can also be written as `nextcord[voice,speed]>=2.0.0a6`)

As per https://github.com/anoadragon453/busty/pull/56#discussion_r793612421, I think we should keep `speed` separate for now, and only recommend it in the documentation if people want a speed boost.

Reply:
I'm fine with keeping speed separate, though my guess is that it's not separate because the required libraries are widely unsupported but instead because if you install nextcord without [voice] or [speed] it becomes dependency-less, which makes it seem cool and clean.

Reply:
Actually, I might be wrong on this. it seems [aiodns](https://pypi.org/project/aiodns/) does not list OSX as a target platform, whereas all the other libraries do.

Reply:
The history behind these options seems to be that discord.py just added the extra `speed` option as a lever to switch between Python's standard json lib and `orjson`: https://github.com/Rapptz/discord.py/commit/88d825a803917ab3e0e9797df9f47be7ed79bb9c. Not much public discussion happened here - it was just pushed to master.

nextcord then added the extra libraries in https://github.com/nextcord/nextcord/pull/78. `Brotli` at least is just python bindings, and depends on a `brotli` package being installed on the system.